docs/_docs/httpbody.md

```diff
@@ -0,0 +1,35 @@
+---
+category: documentation
+---
+
+# HttpBody message Feature
+The [HTTP Body](https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto) messages allows a response message to be specified with custom data content and a custom content type header.
```

Comment:
`The values included in the HTTPBody response will be used verbatim in the returned message from the gateway. Make sure you format your response carefully!`.